front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -44,6 +44,18 @@ class PipelineController {
     pipelineDAO.getPipelinesByApplication(application)
   }
 
+  @RequestMapping(value = 'getById/{id:.+}', method = RequestMethod.GET)
```

Comment:
We can totally move that history URL ... /pipelines/{application}/{id}/history would make sense I suppose.

That `deleteById` is equally bad (_but since day one I suspect_) ... it'd be great to have DELETE /pipelines/{application}/{id}.